react/src/pages/admin/courses/CourseManagementPage.jsx

```
@@ -0,0 +1,79 @@
+                        <th scope="col">#</th>
+                        <th scope="col">Title</th>
+                        <th scope="col">Actions</th>
+                    </tr>
+                </thead>
+                <tbody>
+                    {courses.map((course, index) => (
+                        <tr key={course.id}>
+                            <td scope="row">{index + 1}</td>
+                            <td>{course.title}</td>
+                            <td className="text-end">
+                                <button onClick={() => handleView(course.id)} className="btn btn-info btn-sm me-2">View</button>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)